extensions.json

```diff
@@ -68,5 +68,19 @@
     "sha256sum": "f03d7a517ba6f7d57564eec965358055c87285a8a504d5488af969767a76c4ab",
     "foxe": "https://github.com/danclapp4/ros2-parameter-extension/releases/download/v0.0.1/danielclapp.ros2-parameter-extension-0.0.1.foxe",
     "keywords": ["ROS2", "Parameters", "Param"]
+  },
+  {
+    "id": "foxglove.studio-extension-nipple",
```

Comment:
The `foxglove.` id is reserved for extensions published by foxglove. Please change this to `yulong88888` or `yulong` whichever you prefer.